ci-operator/step-registry/rosa/cluster/wait-ready/nodes/rosa-cluster-wait-ready-nodes-commands.sh

```diff
@@ -268,3 +268,4 @@ else
     exit 1
 fi
 cat "${SHARED_DIR}/cluster-config"
+sleep 14400
```

Comment:
For debugging only?

Reply:
yes... This time window allows me to some log analysis and other checkpoints.